src/vis_cpu/conversions.py

```diff
@@ -52,7 +59,8 @@ def enu_to_az_za(enu_e, enu_n, orientation="astropy"):
     # Flip and rotate azimuth coordinate if uvbeam convention is used
     if orientation == "uvbeam":
         az = 0.5 * np.pi - az
-
+    if constraint_az_between_zero_twopi:
```

Comment:
Is there any reason why we shouldn't just leave this on by default? I don't see any big issues with forcing Azimuth to be in this range.

Reply:
I agree with making the default range 0-2pi (its much more sensible). Off the bat, I didn't want to change the default behavior without consultation but if others are for it I think we should.